stdlib/public/TensorFlow/Ops.swift

```diff
@@ -1292,6 +1292,19 @@ public extension Tensor where Scalar : Numeric {
     return product(squeezingAxes: axes)
   }
 
+  /// Returns the arithmetic mean along the specified axes. The reduced
+  /// dimensions are retained with value 1.
+  /// - Parameter axes: The dimensions to reduce.
+  /// - Precondition: Each value in `axes` must be in the range `-rank..<rank`.
+  @inlinable @inline(__always)
+  @differentiable(
+    wrt: self, vjp: _vjpMean(alongAxes:)
+    where Scalar : TensorFlowFloatingPoint
+  )
+  func mean(alongAxes axes: Tensor<Int32>) -> Tensor {
```

Comment:
@rxwei, would it make sense to change mean(alongAxes axes: [Int32]) to call this directly?

Reply:
Weighing in: yes, I think that would make sense.

Reply:
Done.

Reply:
Yep, Dan's right.